src/components/Clock/Clock.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Clock extends React.Component {
+  state = {
+    time: new Date().toLocaleTimeString(),
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      this.setState({
+        time: (new Date()).toLocaleTimeString(),
+      });
+      // eslint-disable-next-line no-console
+      console.log(new Date().toLocaleTimeString());
+    }, 1000);
+  }
```

Comment:
you also need to clear the timer when the Clock is removed from the DOM, cause now the clock is still ticking in the console event after the Clock component is destroyed
hint: use componentWillMount and clearInterval
https://reactjs.org/docs/state-and-lifecycle.html

Reply:
also, let's create a separate method for updating the time (we can call it 'tick')